sdks/python/setup.py

```diff
@@ -213,22 +242,9 @@ def get_portability_package_data():
               *get_portability_package_data()
           ]
       },
-      ext_modules=cythonize([
-          'apache_beam/**/*.pyx',
-          'apache_beam/coders/coder_impl.py',
-          'apache_beam/metrics/cells.py',
-          'apache_beam/metrics/execution.py',
-          'apache_beam/runners/common.py',
-          'apache_beam/runners/worker/logger.py',
-          'apache_beam/runners/worker/opcounters.py',
-          'apache_beam/runners/worker/operations.py',
-          'apache_beam/transforms/cy_combiners.py',
-          'apache_beam/transforms/stats.py',
-          'apache_beam/utils/counters.py',
-          'apache_beam/utils/windowed_value.py',
-      ],
-                            language_level=3),
+      ext_modules=extensions,
       install_requires=[
+          'build>=0.9.0,<0.11.0',
```

Comment:
interesting. is it common to add build as a dependency ? Also why not 1.x ?

Reply:
Since we use it in stager.py, I added it so that it won't cause a breaking change for the users. 

It seems 1.x was released recently. I will change the bounds. Thanks for noticing. I made this commit a long time back :p

Reply:
i think it should at most be an optional dependency, like snappy